tools/es2015-preset.js

```diff
@@ -1,12 +1,13 @@
-const buildPreset = require('babel-preset-es2015').buildPreset
-
-const BABEL_ENV = process.env.BABEL_ENV
-
-module.exports = {
-  presets: [
-    [ buildPreset, {
-      loose: true,
-      modules: BABEL_ENV === 'es' ? false : 'commonjs'
-    } ]
-  ]
-}
+module.exports = function(api) {
```

Comment:
Can we rename this file to `env-preset` instead of `es2015-preset` to align it with the change of the base preset?